src/platform/silabs/rs911x/BLEManagerImpl.cpp

```diff
@@ -92,6 +107,8 @@ void sl_ble_init()
 
     //  initializing the application events map
     rsi_ble_app_init_events();
+    sl_get_bluethooth_addr(addr, RSI_BLE_ADDR_MAX_LENGTH);
+    rsi_ble_set_random_address_with_value(addr);
```

Comment:
why use `rsi_ble_set_random_address_with_value ` when `rsi_ble_set_random_address` seems simpler?

Reply:
rsi_ble_set_random_address  is the API that will set the static random address but according to matter the BLE address will change at least after every BOOT

Reply:
Are you implying that the addresse set after calling `rsi_ble_set_random_address` will always be the same after a reboot?

Reply:
yes

Reply:
Will take it with the Bluetooth team